_includes/manuals/1.16/5.8.1_database.md

```diff
@@ -1,2 +1,2 @@
 
-All Foreman instances in a cluster/group must point to the same database. This can be done during the initial installation (through flags or altering `foreman_installer_answers.yaml`) or by directly altering `/etc/foreman/database.yaml` and pointing the correct environment (usually production) to your Foreman DB, then restarting Foreman.
+All Foreman instances in a cluster/group must point to the same database. This can be done during the initial installation (through flags or altering `/etc/foreman-installer/scenarios.d/foreman-answers.yaml`) or by directly altering `/etc/foreman/database.yaml` and pointing the correct environment (usually production) to your Foreman DB, then restarting Foreman.
```

Comment:
I'm not sure we should recommend changing the answers file at all - flags should be much more stable and work on multiple scenario's.

Reply:
I'm not sure it's fair to say the installer flag are more stable, since they are created based on the parameters name, just like the parameters in the answer file.

I like to manually edit the answer file when I'm changing multiple values as I find that easier, maybe that's just me.

Reply:
Sure, i get what you saying 👍 updated. Thanks @ekohl